frontend/src/firebase/firebaseInteractor.ts

```diff
@@ -2,7 +2,7 @@ import * as firebase from "firebase/app";
 import "firebase/auth";
 import "firebase/firestore";
 import "firebase/storage";
-import { AccountType, User, Word, Assessment } from "../models/types";
+import { AccountType, Activity1, Activity2, Activity3, Activity4, Assessment, Interventions, InterventionWord, User, Word } from "../models/types";
```

Comment:
Can we rename activity1 -> activity 4 to Definition, Example, Context, and Review

Reply:
Yes!